### DISCUSSION TITLE
Windows Notification App-Title

### DISCUSSION THREAD

**Discussion by BlauCrud** [ncruces/zenity#37] [Q&A] [answered]
Hi there,

first of all, thank you for creating this library. I have a question about Windows notifications. Is it currently possible to change the app title? Or at least to "beautify" it, for example: Microsoft Explorer Notification instead of Microsoft.Explorer.Notification.{...}? 

And is it planned to change the look of the notifications and to clear the notifcation history at a later time?

Thanks in advance and have a nice day!

**Comment by ncruces** [comment]
I don't have any specific plans to improve this. The code I'm using for this is [here](https://github.com/ncruces/zenity/blob/master/notify_windows.go).
I don't think it's possible to do much better using the current strategy.

But I'm open to suggestions, and happy to review pull requests with improved functionalty.

**Comment by ncruces** [comment]
Documentation for [`Shell_NotifyIcon`](https://learn.microsoft.com/en-us/windows/win32/api/shellapi/nf-shellapi-shell_notifyiconw) and [`NOTIFYICONDATA`](https://learn.microsoft.com/en-us/windows/win32/api/shellapi/ns-shellapi-notifyicondataw).

**Comment by ncruces** [comment, marked as answer]
This should be fixed by 39ff6de6ef40ead5a2fed7bca2b8328e49a0f437.
